bchrpc/pb/bchrpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.25.0
-// 	protoc        v3.12.3
+// 	protoc-gen-go v1.22.0
```

Comment:
Bit worried about downgrading the versoin of protoc we are using to generate these. Can we bump protoc and regen?

Reply:
done.